Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM golang:alpine as builder
 MAINTAINER Jessica Frazelle <jess@linux.com>
 
+# Comment
```

Comment:
since it doesn't add value, I wouldn't add it.

Reply:
It was to test the script will remove it

Reply:
yes, instead having tests will be better.

This is dockerfile of the command and this comment looks suspicious.